Program.cs

```diff
@@ -81,38 +52,86 @@ static Statistics StartMenu()
                         citiesStats = new Statistics("Canadacities.csv", DataModeler.SupportedFileTypes.CSV);
                         valid = true;
                         break;
-                    case "exit":
-                        System.Environment.Exit(0);
-                        break;
-                    case "restart":
-                        break;
                     default:
                         Console.Write("\nInvalid selection, please enter a valid selection: ");
                         break;
                 }
             } while (!valid);
             return citiesStats;
         }
+
+        /// <summary>
+        /// Display a main menu to allow users to go to a province menu or city menu
+        /// </summary>
+        /// <param name="cityStats"></param>
+        static void MainMenu(Statistics cityStats)
+        {
+            string selection;
+            string titleText = "Mode Selection Menu";
+            string dash = new string('-', titleText.Length);
+
+            bool displayMenu = true;
+            do
+            {
+                if (displayMenu)
+                {
+                    Console.Clear();
+                    Console.WriteLine($"{dash}\n{titleText}\n{dash}\n");
+                    Console.WriteLine("\nAvailable selections:");
+                    Console.WriteLine("\t1) Display city statistics menu.");
+                    Console.WriteLine("\t2) Display province statistics menu.");
+                    Console.WriteLine("\texit - Exit the program");
+                    Console.Write("Please make a selection: ");
+                }
+                displayMenu = true;
+                selection = Console.ReadLine();
+                switch (selection.ToLower())
+                {
+                    case "1":
+                        CityMenu(cityStats);
+                        break;
+                    case "2":
+                        ProvinceMenu(cityStats);
+                        break;
+                    case "exit":
+                        return;
+                    default:
+                        Console.Write("\nInvalid selection, please enter a valid selection: ");
+                        displayMenu = false;
```

Comment:
I think the menu options themselves should be taken out of the do while. Currently this console.write gets immediately console.clear()'ed when the look resets with no output that it failed. I also think overall we don't need to re=report all options on every invalid entry

Reply:
Wasnt planning on puttint it back in the do-while but if you dont then when yoy successfully finish a task (didplay __) then it will show you the menu prompt again